nuxeo-services/nuxeo-platform-filemanager-core/pom.xml

```diff
@@ -68,6 +68,11 @@
       <artifactId>nuxeo-platform-dublincore</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-tag-core</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I couldn't launch the tests because of deploy errors.
it seems you are introducing a cyclic reference so I couldn't use maven